wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -74,9 +71,9 @@ import           Pos.Wallet.Web.Util        (decodeCTypeOrFail, getAccountAddrsO
 ----------------------------------------------------------------------------
 
 getWAddressBalance :: MonadWalletWebMode m => CWAddressMeta -> m Coin
-getWAddressBalance addr =
-    getBalance <=< decodeCTypeOrFail $ cwamId addr
+getWAddressBalance addr = getBalance <=< decodeCTypeOrFail $ cwamId addr
 
+-- BE CAREFUL: this function works for O(number of used and change addresses)
```

Comment:
Minor language nitpick (although I reckon I'm not the most qualified person when it comes to the subject 😀 ) I think "works in" is more appropriate in the context, or even better "This function has complexity O(...)"